src/FacebookAds/Logger/CurlLogger.php

```diff
@@ -81,6 +81,11 @@ class CurlLogger implements LoggerInterface {
    * @param resource $handle
    */
   public function __construct($handle = null) {
+    
```

Comment:
Overall looks good to me, please remove this empty line, thank you!

Reply:
Removed the empty line, thank you!